tests/Aspire.NATS.Net.Tests/AspireNatsClientExtensionsTests.cs

```diff
@@ -1,18 +1,31 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using Aspire.Components.Common.Tests;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
 using Microsoft.Extensions.Hosting;
 using NATS.Client.Core;
+using OpenTelemetry.Trace;
 using Xunit;
 
 namespace Aspire.NATS.Net.Tests;
 
-public class AspireNatsClientExtensionsTests
+public class AspireNatsClientExtensionsTests : IClassFixture<NatsContainerFixture>
 {
+    private const string DefaultConnectionName = "nats";
     private const string ConnectionString = "nats://apire-host:4222";
```

Comment:
This can instead be:
```csharp
    private string ConnectionString => RequiresDockerTheoryAttribute.IsSupported
                                                ? _containerFixture.GetConnectionString()
                                                : "nats://apire-host:4222";
```
.. which would default to the fallback connection string if docker isn't available. And any test that does really need to connect to the service will correctly fail, and others will run even when docker isn't available.